src/core/gui/inputdevices/TouchDrawingInputHandler.cpp

```diff
@@ -32,7 +32,7 @@ auto TouchDrawingInputHandler::handleImpl(InputEvent const& event) -> bool {
 
     // Do we need to end the touch sequence?
     bool mustEnd = event.type == BUTTON_RELEASE_EVENT;
-    mustEnd = mustEnd || event.type == GRAB_BROKEN_EVENT && this->deviceClassPressed;
+    mustEnd = mustEnd || (event.type == GRAB_BROKEN_EVENT && this->deviceClassPressed);
```

Comment:
Might be cleaner as
```suggestion
    mustEnd |= (event.type == GRAB_BROKEN_EVENT && this->deviceClassPressed);
```

Reply:
That doesn't actually compile.

Reply:
Ah, probably should've been single pipe equal. Well, it's a very minor thing--no worries then.